tools/nl/embeddings/finetuning/run.sh

```diff
@@ -13,15 +13,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if [ $# -ne 1 ]; then
-  echo "Usage: $0 <embeddings-size>  # 'small' or 'medium'"
+ERR=true
+if [ "$1" == "-f" ] || [ "$1" == "-a" ]; then
+  if [ "$1" == "-f" ] && [ "$2" != "" ]; then
+    STAGE="final"
+    TUNED_ALT_MODEL="$2"
+    ERR=false
+  elif [ "$1" == "-a" ]; then
+    STAGE="alternatives"
+    TUNED_ALT_MODEL=""
+    ERR=false
+  fi
+fi
```

Comment:
Are these args in the README?

Also, it is somewhat error prone to parse this, so might recommend using getopts (like in https://github.com/datacommonsorg/website/blob/master/run_test.sh#L201-L219) or just assume specific order of args (with optional args at the end).